src/app/helptext/storage/volumes/volume-status.ts

```diff
@@ -0,0 +1,73 @@
+import { T } from '../../../translate-marker';
+
+export default {
+    dialogFormFields: {
+        disk: {
+            placeholder: T('Member disk'),
+            tooltip: T(''),
+        },
+        passphrase: {
+            placeholder: T('Passphrase'),
+            tooltip: T('')
+        },
+        passphrase2: {
+            placeholder: T('Confirm Passphrase'),
+            tooltip: T(''),
+        },
+        force: {
+            placeholder: T('Force'),
+            tooltip: T(''),
```

Comment:
Set to override safety checks and add the disk to the pool. WARNING: any data stored on the disk will be erased!